nextflow_schema.json

```diff
@@ -479,16 +479,24 @@
                     "hidden": true,
                     "help_text": "Using this params you can add custom args to VEP."
                 },
+                "use_annotation_cache_keys": {
+                    "type": "boolean",
+                    "fa_icon": "fas fa-toolbox",
+                    "description": "Use annotation cache keys for snpeff_cache and vep_cache.",
```

Comment:
can we get a help text with a bit more explanation

Reply:
like i am unsure now, if this referes to the new s3 bucket or if I want to define some cache locally or so

Reply:
yeah, basically that's because of how snpeff and vep store data, they're made for HPC, and I'm storing on cloud.
Plan would be to extrapolate that in the docs.
But I need to build a website for that.

Reply:
ok awesome :)